apps/examples/aifw_test/SineWaveAIModel.h

```diff
@@ -29,11 +29,34 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
-==============================================================================*/ 
-#ifndef __lwaifw_sine_test_model_h__
-#define __lwaifw_sine_test_model_h__
+==============================================================================*/
+
+#pragma once
+#include "aifw/aifw.h"
+static const char sine_wave_version[] = "123456";
+extern const unsigned char sine_wave_model[];
+
+static const AIModelAttribute gSineWaveModelAttribute = {
+    12345, /* crc32 */
```

Comment:
You should follow TizenRT coding rules.
You should use a tab for indentation.

Reply:
TizenRT coding style applied to all files.